programs/util/syslog2stderr.c

```diff
@@ -0,0 +1,41 @@
+#include <dlfcn.h>
```

Comment:
I'm not sure if this program really belongs in our repository. It's there purely for use with the SoftHSM library. We test with SoftHSM, but we don't have a dependency on it as such: the code should work with any PKCS#11 implementation.

That being said, we need this available on the CI. Having it in the mbedtls repository is the easiest way to achieve that.

Reply:
Perhaps put it in `programs/test` then? `programs/utils` is for things that might be useful to end-users, but I think `programs/test` is OK for things we only use for testing.

Reply:
Done.